_posts/2020-04-28-US-RSE-progress.md

```diff
@@ -12,7 +12,9 @@ The US-RSE association was first planned in 2018, but it really became active an
 
 **Membership**. In January 2019, we had 10 members, who were all involved in starting up US-RSE. By the end of 2019, this had grown to 259, and is now (at the end of March 2020) 345.
 
-<p align=center>![US-RSE Membership]({{ site.baseurl }}/assets/posts/2020-04-28-us-rse-membership.png "US-RSE Membership")<br/>_US-RSE Membership_</p>
+<div style="align:center">
```

Comment:
If this doesn't work, usually you can do `margin:auto` to get center alignment for an image. A more robust example is at https://www.w3schools.com/howto/howto_css_image_center.asp.

Reply:
it's now good enough...